apps/web/src/pages/templates/components/TranslateProductLead.tsx

```diff
@@ -10,7 +10,7 @@ export const TranslateProductLead = ({ id, style = {} }: { id: string; style?: C
       title="Translation management"
```

Comment:
Could not get the column variant work with the text for this one so change variant see screenshots for how it looks. To me it looks good.